docs/.vitepress/config.ts

```diff
@@ -56,6 +56,7 @@ export default defineConfig({
     localeLinks: {
       text: 'English',
       items: [
+        { text: 'Español', link: 'https://es.vitejs.dev' },
         { text: '简体中文', link: 'https://cn.vitejs.dev' },
         { text: '日本語', link: 'https://ja.vitejs.dev' }
```

Comment:
Should this be ordered following https://v2.vuejs.org (third option)? I don't quite remember the rules for ordering them.

Reply:
Should be ordered now as requested.